### DISCUSSION TITLE
Do is out of date comments on containerd at branch release 1.25

### DISCUSSION THREAD

**Discussion by xiaods** [k3s-io/k3s#8335] [Q&A] [answered]
**Environmental Info:**
K3s Version:  1.25

Node(s) CPU architecture, OS, and Version: 

Cluster Configuration:

**Describe the bug:**

for scripts/version.sh
```
# We're building k3s against containerd 1.5 in go.mod because 1.6 has dependency
# conflicts with Kubernetes, but we still need to bundle containerd 1.6.
VERSION_CONTAINERD="v1.7.3-k3s1"
```
because containerd verseion already 1.7.3, so the comments is out of date? 

**Steps To Reproduce:**

reading source code.

**Expected behavior:**

remove out of date go package for containerd.

**Actual behavior:**

**Additional context / logs:**

**Comment by brandond** [comment, marked as answer]
Yep. Comments haven't been bumped alongside versions, but the reasoning is still valid.

**Reply by xiaods** [reply]
Thanks @brandond
